PlatformConfig.mk

```diff
@@ -27,7 +27,6 @@ BOARD_KERNEL_PAGESIZE    := 4096
 BOARD_KERNEL_TAGS_OFFSET := 0x00000100
 BOARD_RAMDISK_OFFSET     := 0x01000000
 
-BOARD_KERNEL_CMDLINE += lpm_levels.sleep_disabled=1
 BOARD_KERNEL_CMDLINE += androidboot.bootdevice=4804000.ufshc
```

Comment:
What changed that they were disabled before?

Reply:
that cmdline shoudl be set by dts